<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="trailnet_prediction_node" pkg="trailnet_pytorch" type="live_trailnet_predict.py" output="screen" required="true">
        <rosparam file="$(find trailnet_pytorch)/config/trailnet_params.yaml" command="load" />
    </node>
</launch>